### ISSUE TITLE
shieldui/shieldui-lite instrall

### ISSUE THREAD

**Issue by brockclayton403** [jspm/registry#469]
When I try to install this package fro github, it says the install is complete, but it does not write the files to the jspm_packages/jquery directory.

Here is the messages I get when installing.

     Looking up github:shieldui/shieldui-lite
     Updating registry cache...
ok   Up to date - shieldui/shieldui-lite as github:shieldui/shieldui-lite@master (master)
ok   Install tree has no forks.

ok   Install complete.

**Comment by guybedford** [comment]
The files would be in the `jspm_packages/github/shieldui/shieldui-lite@amster` directory here.

**Comment by brockclayton403** [comment]
Yes, you are right, I don't know how I missed that. The file jspm_packages/github/shieldui/shieldui-lite@master.js was missing from the jspm_packages/github/shieldui/ folder. I added it based on another package file that was in the system. the Aurelia app seems to like things now.

Thanks Guy.
